src/realmGeol.ttl

```diff
@@ -57,307 +288,1624 @@ sorelsp:hasUpperBoundary rdf:type owl:ObjectProperty .
+                  owl:equivalentClass soreag:UpperMantleRigid ;
+                  rdfs:subClassOf soreag:UpperMantle ;
+                  rdfs:label "mesosphere"@en .
+
+
+###  http://sweetontology.net/realmGeol/MineralProvince
+soreag:MineralProvince rdf:type owl:Class ;
+                       rdfs:subClassOf soreag:GeologicProvince ;
+                       dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                       dcterms:source <http://www.glossaryofgeology.org> ;
+                       skos:altLabel "Metallogenic province"@en ;
+                       skos:definition "An area characterized by a particular assemblage of mineral deposits, or by one or more characteristic types of mineralization. A metallogenic province may have had more than one episode of mineralization.  (Neuendorf et al., 2011)"@en ;
+                       skos:prefLabel "mineral province"@en .
 
 
 ###  http://sweetontology.net/realmGeol/Moho
 soreag:Moho rdf:type owl:Class ;
-          owl:equivalentClass soreag:Mohorovi_i_Discontinuity ;
-          rdfs:subClassOf soreag:GeologicBoundary ;
-          rdfs:label "moho"@en .
+            owl:equivalentClass soreag:Mohorovi_i_Discontinuity ;
+            rdfs:subClassOf soreag:GeologicBoundary ;
+            rdfs:label "moho"@en .
 
 
 ###  http://sweetontology.net/realmGeol/Mohorovi_i_Discontinuity
 soreag:Mohorovi_i_Discontinuity rdf:type owl:Class ;
-                              rdfs:label "mohorovicic discontinuity"@en .
+                                rdfs:subClassOf soreag:GeologicBoundary ;
+                                rdfs:label "mohorovicic discontinuity"@en .
+
+
+###  http://sweetontology.net/realmGeol/Monocline
+soreag:Monocline rdf:type owl:Class ;
+                 rdfs:subClassOf soreag:StructuralFeature ;
+                 dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                 dcterms:source <https://doi.org/10.1093/acref/9780199653065.001.0001> ;
+                 skos:definition "A one-limbed flexure on either side of which the strata are horizontal or dip uniformly at low angles. (Allaby, 2013)"@en ;
+                 skos:prefLabel "monocline"@en .
+
+
+###  http://sweetontology.net/realmGeol/MyloniteZone
+soreag:MyloniteZone rdf:type owl:Class ;
+                    rdfs:subClassOf soreag:StructuralFeature ;
+                    dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                    dcterms:source <http://www.glossaryofgeology.org> ,
+                                   <https://doi.org/10.1093/acref/9780199653065.001.0001> ;
+                    skos:definition "Usually found in narrow, planar zones of localised ductile deformation, although kilometre-scale zones exist. Mylonites form in shear zones by mechanical grinding, crushing and recrystallization of rock to produce a metamorphic rock that has foliation and is much finer grained than its precursor. (Modified from: Neuendorf et al., 2011; Allaby, 2013)"@en ;
```

Comment:
suggest def:
A ductile shear zone (link to def--have to add subtype of shear zone) in which the deformed rocks are mylonitic rock (link to def).

subclass of shearzone/ductileshearzone


Scope note -- rest of the text  in current definition

Reply:
I worked on a set of vocabularies like this in the context of Dave Soller's US National Geologic database project a while back.  A couple of observations-- Definitions in the AGI Glossary are for geologists, NOT for information modeling. They are variously circular, ambiguous, or just not definitions (an X is an X...).  Releasing the glossary we generated for that project got hung up on intellectual property questions-- AGI glossaries are copyrighted, and there were questions about the legality of using their definitions and attaching URIs, making them publicly available.  The Glossary (at that time) was a cash-cow for AGI, and AFAIK they haven't produced an online public version yet with URIs for the terms in the glossary. 

I haven't gotten all the way through Nick's contributions-- a massive effort, kudos!.  It brings up the fundamental question again-- is SWEET supposed to be a glossary with 'fuzzy',  geologist-friendly definitions (or none at all), or is it supposed to be an ontology that can be used for semantically precise annotation of data?   If the first, ignore most of my comments. If the second--there's a LOT of work that needs to be done.